test/unit_tests/ZapIn.test.ts

```
@@ -0,0 +1,451 @@
+          "2000000",
+          "10",
+          result.encodedCallData,
+          result.encodedCallData
+        );
+
+      const postBalance = await arrakisV2.balanceOf(userAddress);
+
+      console.log("prevBalance", prevBalance.toString());
+      console.log("postBalance", postBalance.toString());
+    });
+    it.only("#0: Zap out with USDC", async () => {
```

Biome

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)